docs/guides/09-postman-collection.md

```diff
@@ -42,23 +42,23 @@ There are some known limitations that it's important to keep in mind:
 
 ### Authentication
 
-Postman supports a few authentication schemes which OpenAPI does not, such as Hawk and AWS. In this case Prism will do a simplified version of security validation, and just check you've got the right headers populated.
+Postman supports a few authentication schemes that OpenAPI doesn't, such as Hawk and AWS. In this case, Prism will do a simplified version of security validation and verify that you've got the right headers populated.
 
 ### Events
 
-Events are linked to the Postman hosted platform, and so Prism is going to skip this section entirely.
+Events are linked to the Postman hosted platform, so Prism will skip this section entirely.
 
-For this reason, if some of these event handlers have scripts that modify a local variable (globals one are ignored by Prism since they are not tracked in the Postman Application and not the singular Postman Collection) used in some of the request/response pair, Prism won't be aware of these changes.
+For this reason, if some event handlers have scripts that modify a local variable used in a request/response pair, Prism won't be aware of these changes. Global variables are ignored by Prism since they're not tracked in the Postman Application and not the singular Postman Collection.
```

Comment:
Is this part of the sentence right? "since they're not tracked in the Postman Application and not the singular Postman Collection."

You didn't make any changes to that part of the sentence, it just doesn't sound right.

Reply:
@daniel-white : Do you have any thoughts on this? I can change the sentence to:

Global variables are ignored by Prism since they're not tracked in the Postman Application or the singular Postman Collection.

Which makes grammatical sense, but I'm not sure it's technically accurate.